ROADMAP.md

```diff
@@ -16,51 +16,77 @@ new Federation 2 (on the [main branch](https://github.com/apollographql/federati
 ## Table of Contents
 
 * [What's next](#whats-next)
-  * [Federation 2 GA](#federation-2-ga)
-  * [Federation 2 Post-GA](#federation-2-post-ga)
+  * [Next Release](#next-release)
   * [Under Consideration](#under-consideration)
 
 * [Released](#released)
+  * [Federation 2 GA](#federation-2-ga)
   * [Federation 2 Preview](#federation-2-preview)
   * [Federation 2 Alpha](#federation-2-alpha)
-  * [Gateway Performance Enhancements](#gateway-performance-enhancements)
+  * [Gateway Enhancements](#gateway-enhancements)
   * [Federation 1](#federation-1)
   * [Subgraph Compatibility Test Results](#subgraph-compatibility-test-results)
 
 ## What's Next
 
-### Federation 2 GA
+### Next Release
 
-* Incorporate community feedback on Preview
-* Type merging that can be relaxed even further with:
-  * `@inaccessible` - [#1178](https://github.com/apollographql/federation/issues/1178) and [core feature spec](https://specs.apollo.dev/)
-* Field migration across subgraphs with `@override` - [#1177](https://github.com/apollographql/federation/issues/1177)
-* Updated [Fed 2 docs](https://www.apollographql.com/docs/federation/v2/) & [migration guide](https://www.apollographql.com/docs/federation/v2/federation-2/moving-to-federation-2)
-* Enhanced test automation
+* `@tag` export to API schema
+* Compose user-defined directives in subgraphs
 
-### Federation 2 Post-GA
+### Under Consideration
 
-* Compose user-defined directives in subgraphs
+* `@defer` support in query planning
+* Subscriptions support in query planning
+* Entity interfaces can be spread across multiple subgraphs & interface queries with `@interfaceObject` helper.
 * Harmonizing shared value types across subgraphs to a canonical desired state.
 * Advanced caching, auth, demand control, rate limiting, governance, and more!
 * Importing shared types into subgraph schemas, to keep things more DRY.
-* `Entity interfaces` can be spread across multiple subgraphs & interface queries.
 * Nested `@provides` support beyond what Fed 2 alpha already supports natively.
```

Comment:
nit: should we drop alpha from this sentence?

Reply:
good catch, fixed 👍